src/OpenApi/src/Services/NamedService.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace Microsoft.AspNetCore.OpenApi;
+
+// Keyed services don't provide an accessible API for resolving
```

Comment:
I'm a little curious why this isn't a doc comment.  Because it doesn't seem useful in Quick Info?

Reply:
I tend to use standard doc comments for developer-focused notes. But you've reminded me that the benefit of doc comments (even on internal classes) is they show up in Quick Info and I think the comment here would be helpful to see as devs are reading the implementation of `AddOpenApi` where this NamedService is used.